agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -243,6 +248,38 @@ public AzureMonitorExporterBuilder credential(TokenCredential credential) {
    *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
    */
   public AzureMonitorTraceExporter buildTraceExporter() {
+    initExpoterBuilder();
+    return new AzureMonitorTraceExporter(
+        true,
+        builder -> {
+          builder.setInstrumentationKey(instrumentationKey);
+          builder.addTag(
+              ContextTagKeys.AI_INTERNAL_SDK_VERSION.toString(), VersionGenerator.getSdkVersion());
+        },
+        (event, instrumentationName) -> false,
+        telemetryItemExporter::send,
+        telemetryItemExporter::flush,
+        telemetryItemExporter::shutdown,
+        () -> true,
+        () -> null);
+  }
+
+  /**
+   * Creates an {@link AzureMonitorMetricExporter} based on the options set in the builder. This
+   * exporter is an implementation of OpenTelemetry {@link MetricExporter}.
+   *
+   * @return An instance of {@link AzureMonitorMetricExporter}.
+   * @throws NullPointerException if the connection string is not set on this builder or if the
+   *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
+   */
+  public AzureMonitorMetricExporter buildMetricExporter(
+      Consumer<AbstractTelemetryBuilder> telemetryInitializer) {
+    initExpoterBuilder();
```

Comment:
is it a problem if user calls both `buildTraceExporter` and `buildMetricExporter` one after the other? how about if they call `buildTraceExporter` and then update some things (e.g. connectionString) and then they call `buildMetricExporter`?

it's probably worth adding tests for these scenarios as I think we want to support them